workflows/prognostic_c48_run/runtime/transformers/fv3fit.py

```diff
@@ -36,31 +35,67 @@ class Config:
     online: bool = True
 
 
+class TendencyOrStateMultiModelAdapter:
```

Comment:
Would be nice to isolate `TendencyOrState` and `MultiModel` into separate abstrations. These are independent concerns

Reply:
Yeah the MultiModel part had no need to be in here. I've modified implementation to focus on the "inversion" of a dataset that is keyed on ML output names to one that is keyed on the target state variable names (and which has done the sum over tendencies if necessary).